File: SmsFeedback_Take4/Models/Helpers/Agent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SmsFeedback_Take4.Models.Helpers
{
    public class Agent
    {
        public String user;
        public int priority;
        
        public Agent(String iUser, int iPriority)
        {
            user = iUser;
            priority = iPriority;
        }
    }
}

File: Moderator/src/org/helpers/json/Agent.java
package org.helpers.json;
import java.util.*;

public class Agent {
	public String user;
	public int priority;
	public ArrayList<Device> devices;
	
	public Agent(String iUser, ArrayList iDevices, int iPriority) {
		user = iUser;
		devices = iDevices;
		priority = iPriority;
	}
	
	public String getUser() {
		return user;
	}

	public void setUser(String user) {
		this.user = user;
	}

	public int getPriority() {
		return priority;
	}

	public void setPriority(int priority) {
		this.priority = priority;
	}
}
